memgpt/cli/cli.py

```diff
@@ -38,6 +38,7 @@ def run(
     human: str = typer.Option(None, help="Specify human"),
     model: str = typer.Option(None, help="Specify the LLM model"),
     preset: str = typer.Option(None, help="Specify preset"),
+    prompt: str = typer.Option(None, help="Specify prompt"),
```

Comment:
This should probably say "Specify system prompt"

Reply:
Done.